app/src/test/java/com/mstudio/superheroarch/repository/RickAndMortyRepositoryTest.kt

```diff
@@ -60,4 +64,27 @@ class RickAndMortyRepositoryTest {
         `when`(rickAndMortyApiMock.getSingleEpisode(1)).thenThrow(Exception())
         rickAndMortyRepository.getSingleEpisode(1)
     }
+
+    @Test
+    fun `given rickAndMorty repository, when set character as favorite, then update user as favorite in database`() = runTest {
+        val result = rickAndMortyRepository.updateFavoriteCharacterStatus(true, 1)
+        val localCharacters = rickAndMortyRepository.getCharacters()
```

Comment:
I'll add a previous getCharacters checking that isFavorite is false before doing the updateFavoriteCharacterStatus